docs/intro/tutorial.rst

```diff
@@ -460,19 +516,24 @@ Here is a modification to our spider that does just that::
         ]
 
         def parse(self, response):
-            for quote in response.xpath('//div[@class="quote"]'):
-                item = QuoteItem()
-                item['text'] = quote.xpath('span[@class="text"]/text()').extract_first()
-                item['author'] = quote.xpath('span/small/text()').extract_first()
-                yield item
-            next_page = response.xpath('//li[@class="next"]/a/@href').extract_first()
-            if next_page:
+            for quote in response.css('div.quote'):
+                yield {
+                    'text': quote.css('span.text::text').extract_first(),
+                    'author': quote.css('span small::text').extract_first(),
+                    'tags': quote.css("div.tags a.tag::text").extract(),
```

Comment:
A very minor nitpick: probably it is better to use single quotes here, for consistency with other selectors.